### PR TITLE
Revolt support

### PR DESCRIPTION
**Pull request by showierdata9978**

### CODE REVIEW

**Comment by JoshAtticus** [comment]
When you hover over the bridged tag, it says the message is from a discord server. You should change it to say something like “This message is from an external platform”

**Comment by JoshAtticus** [comment]
Pfps and profiles are broken

image

**Comment by JoshAtticus** [comment]
@showierdata9978

File: src/lib/Post.svelte [hunk 1]
```diff
@@ -205,7 +205,7 @@
 				{#if bridged}
 					<Badge
 						text="BRIDGED"
-						title="This post is a post bridged from a Discord server by the @Discord bot"
+						title="This post is bridged from a external service by a bot"
```
Comment (williamhorning) [inline src/lib/Post.svelte:208]:
```suggestion
						title="This post is bridged from an external service by a bot"
```

**Review by williamhorning — CHANGES_REQUESTED** [review]
Fix this gramatical error and have Revower profile links actually work before merging

**Comment by JoshAtticus** [comment]
> Fix this grammatical error and have Revower profile links actually work before merging

Fixed @williamhorning

**Review by williamhorning — APPROVED** [review]
@JoshAtticus LGTM then

**Pull request merged**